ReactAndroid/src/main/jni/third-party/folly/Android.mk

```diff
@@ -2,20 +2,35 @@ LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES:= \
-	folly/json.cpp \
-	folly/Unicode.cpp \
-	folly/Conv.cpp \
-	folly/Demangle.cpp \
-  	folly/detail/MallocImpl.cpp \
-  	folly/StringBase.cpp \
-  	folly/dynamic.cpp \
+  folly/json.cpp \
+  folly/Unicode.cpp \
+  folly/Conv.cpp \
+  folly/Demangle.cpp \
+  folly/memory/detail/MallocImpl.cpp \
+  folly/String.cpp \
+  folly/dynamic.cpp \
+  folly/Format.cpp \
+  folly/json_pointer.cpp \
+  folly/lang/ColdClass.cpp \
+  folly/detail/Demangle.cpp \
+  folly/hash/SpookyHashV2.cpp \
+  folly/container/detail/F14Table.cpp \
+  folly/ScopeGuard.cpp \
```

Comment:
Longer term it would be great to figure out a way to not list these out manually, so that upgrades can be easier. I don't know what the best solution is, but perhaps the folly repo has a cmake configuration we can use.